source/Editor.js

```diff
@@ -325,17 +324,23 @@ proto.setSelection = function ( range ) {
         if ( isIOS ) {
             this._win.focus();
         }
-        var sel = this._sel;
-        sel.removeAllRanges();
-        sel.addRange( range );
+        var sel = this._getWindowSelection();
+        if (sel) {
```

Comment:
As per code convention in the rest of the library, there needs to be spaces around sel.